cl/recap/tasks.py

```diff
@@ -897,27 +825,19 @@ def process_case_query_page(self, pk):
             content_updated = True
 
     if pq.debug:
-        mark_pq_successful(pq, d_id=d.pk)
-        self.request.chain = None
+        await mark_pq_successful(pq, d_id=d.pk)
         return {"docket_pk": d.pk, "content_updated": False}
 
     try:
-        d.save()
-        add_bankruptcy_data_to_docket(d, data)
+        await d.asave()
+        await sync_to_async(add_bankruptcy_data_to_docket)(d, data)
     except IntegrityError as exc:
         logger.warning(
             "Race condition experienced while attempting docket save."
         )
         error_message = "Unable to save docket due to IntegrityError."
-        if self.request.retries == self.max_retries:
-            mark_pq_status(pq, error_message, PROCESSING_STATUS.FAILED)
-            self.request.chain = None
-            return None
-        else:
-            mark_pq_status(
-                pq, error_message, PROCESSING_STATUS.QUEUED_FOR_RETRY
-            )
-            raise self.retry(exc=exc)
+        await mark_pq_status(pq, error_message, PROCESSING_STATUS.FAILED)
+        return None
```

Comment:
Another one where there's some semblance of an explanation of why we want retries.

Reply:
retry logic implemented